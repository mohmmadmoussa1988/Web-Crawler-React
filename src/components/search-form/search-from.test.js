import React from 'react';
import {configure,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SearchForm from './search-form';
import toJson from 'enzyme-to-json';
import { Provider } from 'react-redux';
import store from '../../redux/store';

configure({adapter:new Adapter()});
describe(`<SearchForm />`,()=>{

    it('should render SearchForm',()=>{
        const wrapper = shallow(<Provider store={store}><SearchForm /></Provider>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })


})