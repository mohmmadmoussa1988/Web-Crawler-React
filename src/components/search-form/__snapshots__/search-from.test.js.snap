// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> should render SearchForm 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <SearchForm />
</ContextProvider>
`;
